src/GuiBase/Utils/KeyMappingManager.hpp

```diff
@@ -45,6 +51,29 @@ class RA_GUIBASE_API KeyMappingManager : public Ra::Core::Utils::Observable<>
                                                    int key,
                                                    bool wheel = false );
 
+    /// Add a given action to the mapping system.
+    /// This allow to define default behavior when some KeyMappingManageable object is not
+    /// parameterized in the application config file. The action is added to the current config file
+    /// so that it will remain for subsequent usage.
+    /// @todo write the configuration in the configFile to be later reused or modified ?
+    /// @param context the context of the action
+    /// @param keyString  represents the key that need to be pressed to trigger the event
+    /// (ie Key_Z, for example), "" or "-1" or absent correspond to no key needed.
```

Comment:
```suggestion
    /// (ie Key_Z, for example), "" or "-1" or absent corresponds to no key needed.
```
b.t.w. how can it be absent? there is no default value...

Reply:
I still don't get how it can be absent ?